templates/Silk.NET.Templates/templates/silkglgame/Program.cs

```diff
@@ -0,0 +1,41 @@
+using Silk.NET.Windowing;
+using Silk.NET.Input;
+using Silk.NET.OpenGL;
+
+WindowOptions windowOptions = WindowOptions.Default;
+windowOptions.Title = "My Silk.NET Window";
```

Comment:
Could use the `with` syntax, or at least that'd be what I'd recommend to users these days.